ando/tools/generator/AnDOGenerator.py

```diff
@@ -113,6 +113,15 @@ def register_metadata_files(self, *files):
         Parameters
         ----------
         *files: list
+    def generate_metadata_file_tasks(self):
```

Comment:
this method should be equivalent to the other `generate_metadata_file_...` methods, i.e. have an output parameter and only minimal content

Reply:
done

Reply:
I don't see the change here yet even after reloading. Did you push your commit that changes this?

Reply:
ah it's an error in the doc string actually, I'll remove it